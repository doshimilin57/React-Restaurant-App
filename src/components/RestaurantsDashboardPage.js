import React from "react";
import RestaurantList from "./RestaurantList";
import RestaurantListFilters from "./RestaurantListFilters";

const RestaurantsDashboardPage = () => (
  <div>
    <RestaurantListFilters />
    <RestaurantList />
  </div>
);

export default RestaurantsDashboardPage;
