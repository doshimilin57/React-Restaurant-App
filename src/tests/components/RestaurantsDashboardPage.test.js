import React from 'react';
import { shallow } from 'enzyme';
import RestaurantsDashboardPage from '../../components/RestaurantsDashboardPage';

test('should render RestaurantsDashboardPage properly', () => {
  const wrapper = shallow(<RestaurantsDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
