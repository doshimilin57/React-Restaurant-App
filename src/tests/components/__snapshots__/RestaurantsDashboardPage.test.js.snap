// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render RestaurantsDashboardPage properly 1`] = `
<div>
  <Connect(RestaurantListFilters) />
  <Connect(RestaurantList) />
</div>
`;
